Release 5.1.1

ankush-sqops opened the pull request.

ankush-sqops commented:
I have verified the bottlerocket ami creations and node coming up.

manish8607 reviewed (commented):
Please test it once.

ankush-sqops commented:
New releases and fixes:-
- Added bottlerocket AMI support for both arm and amd based architectures.
- Added an option to pass launch template name while creating custom managed AWS node group.
- Added tags on AWS resources to track cost.
